BLSQ/iaso
# IA-2936, IA-2965: update imports from blsq-comp

Explain what problem this PR is resolving

Related JIRA tickets : IA-2936, IA-2965

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

- Remove unused `LinkProvider`
- Replace route utils by imports from blsq-comp (see: https://github.com/BLSQ/bluesquare-components/pull/153)

## How to test

- update package.json to use the correct branch of bluesquare-components

nothing should have changed anywhere...


## Notes
⚠️ Depends on https://github.com/BLSQ/bluesquare-components/pull/153

My linter seemed to be struggling, so some files may not have been linted properly
